### ISSUE TITLE
[0.45.2] Firestarters FS9-C and FS9-P swapped

### ISSUE THREAD

**Issue by Xeno426** [MegaMek/megamek#1199]
### Environment
0.45.2

### Description
The FS9-P variant of the Firestarter in MM has four rocket launcher 10s and two RL-15s with a BV of 940; this is the FS9-C's loadout.
http://www.masterunitlist.info/Unit/Details/1111/firestarter-fs9-c

The actual FS9-P is notable for being the only Firestarter with SRMs (one in each torso), and has a BV of 775. In MM, the mech that fits this bill is the FS9-C.
http://www.masterunitlist.info/Unit/Details/1095/firestarter-fs9-p

Basically, these two units in MM need to have their names swapped.

**Comment by HammerGS** [comment]
Fixed in https://github.com/MegaMek/megamek/commit/cb3bf61b6735258f2dc6e1bea51de4a77e1fc2dc
